etc/pretty.txt

```diff
@@ -17,7 +17,7 @@
           █ ▅▀█▂▁           ■▆▂      ▁▆▘            ███       ███ ▔▀████▀█████  ███ 
            ▜██▎▔▜▄            █     ▄▀                      	                   
           ▃▇▉▔  ▐▀            █  ▃■▀                Pipelines in Genomix,
-        ■▇█▃▂▁ ▗█           ▁▛▀▀▔▔                  © 2021 Akalin Lab
+        ■▇█▃▂▁ ▗█           ▁▛▀▀▔▔                  © 2022 Akalin Lab
```

Comment:
nitpick: this should be 2021-2022 instead of just the last year.

Reply:
Fixed now, will merge then...